packages/commons/src/AxeResultNode.cs

```diff
@@ -3,16 +3,50 @@
 
 namespace Deque.AxeCore.Commons
 {
+    /// <summary>
+    /// A tested rule element.
+    /// </summary>
     public class AxeResultNode
     {
+        /// <summary>
+        /// Target identifier of this result node.
+        /// </summary>
         [JsonProperty("target", ItemConverterType = typeof(AxeResultTargetConverter), NullValueHandling = NullValueHandling.Ignore)]
         public List<AxeResultTarget> Target { get; set; }
+
+        /// <summary>
+        /// Xpath selectors for elements
```

Comment:
It would be good to clarify that this will only be present if the XPath option was passed as true

Reply:
Actually, nevermind this, it's already addressed in #58